symfony/src/EventListener/RequestListener.php

```diff
@@ -20,11 +30,36 @@ public function onKernelRequest(RequestEvent $event)
             return;
         }
 
+        $this->request = $event->getRequest();
+
         $this->logRequest();
+        if ($fullPageNoticeResponse = $this->checkPageNotices()) {
+            $event->setResponse($fullPageNoticeResponse);
+        }
     }
 
     private function logRequest()
     {
         $this->logger->log(LoggerService::REQUEST);
     }
+
+    private function checkPageNotices()
+    {
+        $path = $this->request->getPathInfo();
+        $activeNotices = $this->em->getRepository(Notice::class)
+            ->getActiveNotices($path);
+
+        // If one of the notices is a full page notice, render the notice response and return
+        if (!preg_match('/^(\/s)?\/admin/', $path)) { // Ignore full page notices for admin urls
```

Comment:
If we ever drop the `/s` post-migration, we'll want to adjust this.

Reply:
The `/s` is optional, so it will match both `/admin...` and `/s/admin...`. But yes, after we completely migrate, we can drop the optional `/s`.

Reply:
:+1: Some day I'll learn how to reed a regex. 😖